documentation/staging/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -86,16 +113,16 @@ The usage of the create script is as follows:
 ```shell
 $ sh create-domain.sh -h
 ```
-```
-usage: create-domain.sh -o dir -i file -u username -p password [-k] [-e] [-h]
+```text
+usage: create-domain.sh -o dir -i file -u username -p password [-m wdt|wlst] [-n encryption-key] [-e] [-v] [-h]
   -i Parameter inputs file, must be specified.
   -o Ouput directory for the generated properties and YAML files, must be specified.
   -u User name used in building the image for WebLogic domain in image.
   -p Password used in building the image for WebLogic domain in image.
+  -m WebLogic configuration mode. Either 'wdt' or 'wlst', optional. Defaults to 'wdt'.
   -e Also create the resources in the generated YAML files, optional.
   -v Validate the existence of persistentVolumeClaim, optional.
-  -k Keep what has been previously cloned from https://github.com/oracle/docker-images.git, optional.
-     If not specified, this script will always remove the existing project and clone again.
+  -n Encryption key for encrypting passwords in the WDT model and properties files, optional.
```

Comment:
Should this be non-optional? Can/should the image build delete the model & properties file when it's done with them?

Reply:
The -n option is for optionally encrypting the passwords in the WDT model and properties file which is supported in WDT mode.

Reply:
I think I understand the purpose. The reason I'm asking is I'm wondering if encryption should be the default, or not even optional, given that having plain-text passwords lying around in files is strongly discouraged...  Alternatively, does the generated image embed the inputs file? If so, it could be deleted so as not to leave it around?

Reply:
I think imagetool removes the domain.properties file after creating the domain, and I can't find it in the resulting image. But need to confirm with @jshum2479

Reply:
There may be a version on the local file system that also needs protection, in which case maybe the encryption is required? In short, we don't want to leave unencrypted plain text passwords on the local file system or in the image...

Reply:
create-domain.sh removes the file on local file system. 

With the -n option and with wdt, customers have an option to encrypt the password in the sample and not have plain text password anywhere.